pytorch_lightning/tuner/lr_finder.py

```diff
@@ -42,34 +42,37 @@
 log = logging.getLogger(__name__)
 
 
+def __determine_lr_attr_name(trainer, model: LightningModule) -> str:
```

Comment:
```suggestion
def _determine_lr_attr_name(trainer, model: LightningModule) -> str:
```
any reason for this?